Dockerfile

```diff
@@ -25,12 +25,13 @@ RUN apt-get update \
   && chown -R app:app $BUNDLE_PATH
 
 # Install chrome and chromedriver for selenium
-ENV CHROME_VERSION 106.0.5249.61
+ENV CHROME_VERSION 119.0.6045.123
+ENV CHROMEDRIVER_VERSION 114.0.5735.90
```

Comment:
Are you able to run the system test using the versions above?

Reply:
I left only CHROME_VERSION and updated it, and the system test ran OK.